operator/pkg/manifest/installer.go

```diff
@@ -426,26 +427,23 @@ func ApplyManifest(componentName name.ComponentName, manifestStr, version, revis
 	// We sort them by namespace so that we can pass the `-n` to the apply command. This is required for prune to work
 	// See https://github.com/kubernetes/kubernetes/issues/87756 for details
 	namespaces, nonNsCrdObjectsByNamespace := splitByNamespace(objectsNotInLists(objects, nsObjects, crdObjects))
-	var applyErr error
+	var applyErrors *multierror.Error
 	for _, ns := range namespaces {
 		nonNsCrdObjects := nonNsCrdObjectsByNamespace[ns]
 		nsOpts := opts
 		nsOpts.Namespace = ns
 		stdout, stderr, err = applyObjects(nonNsCrdObjects, &nsOpts, stdout, stderr)
 		if err != nil {
-			applyErr = fmt.Errorf("error applying object to %v: %v: %v", ns, err, applyErr)
+			applyErrors = multierror.Append(applyErrors, errors.Wrapf(err, "error applying object to namespace %s", ns))
 		}
 		appliedObjects = append(appliedObjects, nonNsCrdObjects...)
 	}
 	mark := "✔"
-	if applyErr != nil {
+	if applyErrors != nil {
 		mark = "✘"
 	}
 	logAndPrint("%s Finished applying manifest for component %s.", mark, componentName)
-	if applyErr != nil {
```

Comment:
@howardjohn These are redundant calls to `buildComponentApplyOutput`. Simplified